### DISCUSSION TITLE
Configuration file support?

### DISCUSSION THREAD

**Discussion by shaharkazaz** [jscutlery/semver#284] [Q&A] [answered]
Hey!

Does this lib currently support passing options via configuration file within the lib instead of inline options? or even a global configuration? (e.g. the prefix value)

**Comment by edbzn** [comment, marked as answer]
Hi, 

You can also define the options in the workspace.json file:

```json
{
  "executor": "@jscutlery/semver:version",
  "options": {
    "versionTagPrefix": "v"
  }
}
```

We discussed about supporting a dedicated semver configuration file but nothing is planned at the moment. Does it help you?

**Reply by shaharkazaz** [reply]
Yes it does 😄

**Comment by its-miller-time** [comment]
+1 a config file would be game changing...im trying to figure out how to get configs set in a similar way to @semantic-release. I have a single .releaserc.json file that allows an incredible level of configuration.
